tests/simple/testdata/comparisons.textproto

```diff
@@ -1,2 +1,131 @@
 name: "comparisons"
 description: "Tests for boolean-valued functions and operators."
+section {
+  name: "Literal comparison"
+  description: "Comparing literals (no bindings)"
+  test {
+    name: "self_test_equals_int64"
+    expr: "1 == 1"
+    value: { bool_value: true }
+  }
+  test {
+    name: "self_test_not_equals_int64"
+    expr: "-1 == 1"
+    value: { bool_value: false }
+  }
+  test {
+    name: "self_test_equals_uint64"
+    expr: "2u == 2u"
+    value: { bool_value: true }
+  }
+  test {
+    name: "self_test_not_equals_uint64"
+    expr: "1u == 2u"
+    value: { bool_value: false }
+  }
+  test {
+    name: "self_test_equals_double"
+    expr: "1.0 == 1.0e+0"
+    value: { bool_value: true }
+  }
+  test {
+    name: "self_test_not_equals_double"
+    expr: "-1.0 == 1.0"
+    value: { bool_value: false }
+  }
+  test {
+    name: "self_test_equals_string"
```

Comment:
Add a non-empty string equality test like 'abc' == 'abc'  =>  true.
Also add a case-sensitivity test:  'abc' == 'ABC'  =>  false.

Reply:
The first one exists for 'a'=='a'. I'll change it to 'abc', as it's more representative. I'll add a case sensitivity test as well.